File: First_Semester/lab/workspace/work_java/src/b형대비특강/Day_0901/NextLevel.java
package b형대비특강.Day_0901;

import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;

public class NextLevel {

	static class User {
		private String username;
		private int level;
		
		public User() {
			this.username = "codetree";
			this.level = 10;
		}
		
		public User(String username, int level) {
			this.username = username;
			this.level = level;
		}
		public String getUsername() {
			return username;
		}
		public void setUsername(String username) {
			this.username = username;
		}
		public int getLevel() {
			return level;
		}
		public void setLevel(int level) {
			this.level = level;
		}
		
		@Override
		public String toString() {
			return "user " + this.username + " lv " + this.level;
		}
	}
	
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
		String[] temp = br.readLine().split(" ");
		String username = temp[0]; int level = Integer.parseInt(temp[1]);
		User user1 = new User();
		User user2 = new User(username, level);
		bw.write(user1.toString());
		bw.newLine();
		bw.write(user2.toString());
		bw.flush();
		bw.close();
	}

}

File: First_Semester/lab/workspace/work_python/B형 대비 특강/Day_0901/객체/next_level.py
class User():
    def __init__(self,id = 'codetree',level = '10'):
        self.__id = id
        self.__level = level

    def __str__(self):
        return 'user {0} lv {1}'.format(self.__id, self.__level)

    @property
    def id(self):
        return self.__id

    @id.setter
    def set_id(self,id):
        self.__id = id

    @property
    def level(self):
        return self.__level

    @level.setter
    def set_level(self,level):
        self.__level = level

user1 = User()
id, level = input().split()
user2 = User(id, level)
userList = [user1, user2]

for user in userList:
    print(user)
